src/Operation/Aggregate.php

```diff
@@ -272,6 +280,10 @@ private function createCommand(Server $server, $isCursorSupported)
             $cmd['collation'] = (object) $this->options['collation'];
         }
 
+        if (isset($this->options['hint'])) {
+            $cmd['hint'] = $this->options['hint'];
```

Comment:
If `hint` is an array, we should cast it to an object. This hands the edge case of the hint document: `{ "0": 1 }`, which could be `[ "0" => 1]` in PHP and would otherwise convert to a BSON array. If it's not an array (i.e. string or object), we can leave it as-is.

See `Count::createCommand()` for an example.